scripts/spidhub-fargate/README.md

```diff
@@ -0,0 +1,33 @@
+# SPID Hub Fargate
+
+## Requirements
+
+- AWS CLI `brew install awscli`
+- jq `brew install jq`
+- docker `brew install --cask docker`
```

Comment:
Dubbio: questo step installa docker desktop? In caso di risposta affermativa vanno indagate le questioni di licenza